applications/conceptual-model-editor/src/action/center-viewport-to-visual-entity.ts

```diff
@@ -1,18 +1,41 @@
+import { isVisualGroup, isVisualNode, isVisualRelationship } from "@dataspecer/core-v2/visual-model";
 import { ModelGraphContextType } from "../context/model-context";
 import { UseDiagramType } from "../diagram/diagram-hook";
 import { UseNotificationServiceWriterType } from "../notification/notification-service-context";
+import { SemanticModelRelationship } from "@dataspecer/core-v2/semantic-model/concepts";
+import { isSemanticModelRelationshipUsage, SemanticModelRelationshipUsage } from "@dataspecer/core-v2/semantic-model/usage/concepts";
+import { ClassesContextType } from "../context/classes-context";
+import { isAnAttribute, temporaryDomainRangeHelper } from "../util/relationship-utils";
 
+// TODO: Falls apart if we want to represent 1 semantic entities by more visual ones, in such case we want to pass in identifiers of both semantic and the visual entity.
```

Comment:
Move this to an issue. I do not see how this should be resolved localy.

Reply:
Both, I would like to have this as an issues, as we plan to identify entities using model identifier and entity identifier.